pkg/remote/trans/netpollmux/server_handler.go

```diff
@@ -473,3 +497,11 @@ func getRemoteInfo(ri rpcinfo.RPCInfo, conn net.Conn) (string, net.Addr) {
 	}
 	return ri.From().ServiceName(), rAddr
 }
+
+func (t *svrTransHandler) getFirstSvcInfo() (svcInfo *serviceinfo.ServiceInfo) {
```

Comment:
map range is a random behavior.

Reply:
It's ok because there's only one service right now

Reply:
Renamed to `getDefaultSvcInfo()`